travis-ci.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e -x -o pipefail
 
+if [ "${VIBED_SSL:-openssl}" == "botan" ]; then
+    grep -qF "Have_botan" stream/dub.sdl || echo 'versions "Have_botan"' >> stream/dub.sdl
+    # Make botan mandatory (i.e. non-optional)
+    sed -E 's/(dependency "botan".*) optional=true/\1/' -i stream/dub.sdl
+fi
```

Comment:
Yup, at least I couldn't think of one and this one is not too bad: it's only two lines long and even idempotent.

If wanted I could also add an else clause that idempotently resets this change, but imho (1) this script is only used for the CI anyways and (2) if used locally `git checkout` already works well.